src/gui/GuiWindow.zig

```diff
@@ -143,12 +147,19 @@ pub fn mainButtonPressed(self: *const GuiWindow, mousePosition: Vec2f) void {
 	}
 }
 
+pub fn getButtonPositions(self: *const GuiWindow) std.meta.Tuple(&.{ f32, f32, f32 }) {
```

Comment:
Why not just use a `[3]f32` as the return type?

Reply:
Fixed. That was because I was hoping to be able to set multiple variables at once with that return type, like in Python.